### DISCUSSION TITLE
Is it possible to generate and load a spectrogram data from a JSON file ?

### DISCUSSION THREAD

**Discussion by SofYIane** [katspaugh/wavesurfer.js#2770] [Q&A]
I noticed in the documentation that there is a function called 'loadFrequenciesData(url)' that allows us to retrieve frequency data from a JSON file and display it on the canvas. 
However, I'm wondering if there is a similar method to generate spectrogram data, similar to the way 'audiowaveform' does it for waveforms? 
Also, could you please provide information on the expected structure of the JSON file for the frequency data, so I can ensure it's in the correct format?

Thank you !!

**Comment by katspaugh** [comment]
Looks like there's a `getFrequences(callback)` method. Callback receives the JSON.
The code says the JSON is in a "[channel, sample, freq] format". I guess you can call `getFrequences` and see what format it returns.
